cmd/stcrashreceiver/stcrashreceiver.go

```diff
@@ -78,14 +82,42 @@ func (r *crashReceiver) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+// serveGet responds to GET requests by serving the uncompressed report.
+func (r *crashReceiver) serveGet(base string, w http.ResponseWriter, req *http.Request) {
+	path := filepath.Join(r.dirFor(base), base)
+	fullPath := filepath.Join(r.dir, path)
+	if fd, err := os.Open(fullPath + ".gz"); err == nil {
+		// There is a compressed report
+		defer fd.Close()
+		gr, err := gzip.NewReader(fd)
+		if err != nil {
+			http.Error(w, "Internal server error", http.StatusInternalServerError)
+			return
+		}
+		_, _ = io.Copy(w, gr) // best effort
```

Comment:
In theory you could just wedge content-encoding: gzip and return the zipped bytes, but yeah, this shouldn't matter much.

Reply:
Yes possibly, after checking if the client supports it etc but I figured this code is almost never executed to begin with (very few viewers, likely just me) and I couldn't be bothered. :)